community/modules/file-system/gke-persistent-volume/README.md

```diff
@@ -40,6 +41,36 @@ The following example creates a Filestore and then uses the
     use: [datafs-pv, compute_pool]
 ```
 
+The following example creates a GCS bucket and then uses the
+`gke-persistent-volume` module to use the bucket as shared storage in a
+`gke-job-template`.
+
+```yaml
+  - id: gke_cluster
+    source: community/modules/scheduler/gke-cluster
+    use: [network1]
+    settings:
+      master_authorized_networks:
+      - display_name: deployment-machine
+        cidr_block: <your-ip-address>/32
+
+  - id: data-bucket
+    source: community/modules/file-system/cloud-storage-bucket
+    settings: { local_mount: /data }
```

Comment:
nit. Is there a particular reason to use inline syntax. IIUC, unlike sequences, mapping is very picky for correct syntax (spaces) and can be a point of user frustration.

Reply:
Just trying to look cool. Good feedback I have updated.